server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+- select `Webauthn Register` as `Required Action`
+- mark `Enabled` and `Default Action` checkbox
+
+Enable 2FA by Webauthn Authenticator::
+
+An administrator carries out the following operations on Admin Console :
+
+- open `Authentication -> Flows` tab
+- copy and create the new browser flow
+- add `WebAuthn Authenticator` execution below `User Password Form` execution
+- set `OPTIONAL` to `WebAuthn Authenticator` execution's requirement
+- open `Authentication -> Bindings` tab
+- select the created new browser flow as `Browser Flow`
+- push `Save` button
+
+==== Register WebAuthn Authenticator
+
+There are two types of registering a WebAuthn authenticator, for a user who has not yet registered their account on {project_name}, and for a user who has already registered their account on {project_name}.
+
+For the user to identify their registered WebAuthn authenticator afterwards, the user can enter the text as their label.
+
+User without their account::
+
+A user carries out the following operations :
+
+- open the login form
+- click `Register` link
+- fill in items on the register form and push `Register` button
+- the user's browser asks the user to register their WebAuthn authenticator
+- after successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label
+
+User with their account::
+
+At first, an administrator carries out the following operations on `Admin Console` to require the user to register their WebAuthn authenticator after their login :
+
+- open `Users -> (user) -> Details` tab
+- set `WebAuthn Register` on `Required User Actions`
+- push `Save` button
+
+After that, the user carries out the following operations :
+
+- open the login form
+- fill in items and push `Login` button
+- after logged in, the user's browser asks the user to register their WebAuthn authenticator
+- after successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label
+
+==== Authenticate by WebAuthn Authenticator
+
+After registering their WebAuthn authenticator, the user carries out the following operations :
+
+- open the login form
+- fill in the items and push `Login` button
+- the list of registered Webauthn Authenticators' labels is shown. push `Authenticate` button as it is
+- the user's browser asks the user to authenticate by their WebAuthn authenticator
+
+==== View Registered WebAuthn Authenticator
+
+An administrator and a user can view the following information :
+
+- https://www.w3.org/TR/webauthn/#credential-id[Credential ID]
+- Label (WebAuthn authenticator's label the user entered on registering it)
+- https://fidoalliance.org/specs/fido-v2.0-rd-20180702/fido-metadata-statement-v2.0-rd-20180702.html#authenticator-attestation-guid-aaguid-typedef[AAGUID]
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Users -> (user) -> Credentials` tab
```

Comment:
Following the same format as earlier this would be:
Open the `Users -> (user) -> Credentials` tab.

Reply:
Thank you. I'll fix this sentence as you suggested.